frontend/src/api/k8s/servingRuntimes.ts

```diff
@@ -171,8 +181,33 @@ export const assembleServingRuntime = (
     volumes.push(getshmVolume('2Gi'));
   }
 
-  updatedServingRuntime.spec.volumes = volumes;
+  if (nimPVCName) {
```

Comment:
This code should never have been here -- you folks need to work on lowering your points of contact when fixing a blocker fix. You snuck this PVC thing into a high trafficked area for 2 other flows you're not touching instead of just adding this to your serving runtime template before ever calling this flow.

This change has already been tested and I'm going to let it in. But please do not sneak more "nice to have" code in when you need to fix critical blocking issues.